src/components/input/demoBasicUsage/index.html

```diff
@@ -57,7 +58,7 @@
         </md-input-container>
       </div>
 
-      <md-input-container flex>
+      <md-input-container layout="column">
```

Comment:
Is the `layout="column"` necessary here since the only contents are the label and textarea?